examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
<p>gradients are computed separately and then averaged on one device (usually, the first GPU of the available)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>